hypothesize/utilities.py

```diff
@@ -3,6 +3,7 @@
 from scipy.stats import trim_mean
 from scipy.stats import t
 import pandas as pd
+import sys
```

Comment:
We could avoid this import since it is only used here. The error that is raised in your function could instead use `raise Exception("error message here"). What do you think?

Reply:
I agree! Think it was just out of habit haha!